optaplanner-examples/src/test/java/org/optaplanner/examples/nqueens/integration/util/QueenCoordinates.java

```diff
@@ -0,0 +1,25 @@
+package org.optaplanner.examples.nqueens.integration.util;
+
+public class QueenCoordinates  {
+
+    public static QueenCoordinates createQueenCoordinates(int x, int y) {
+        return new QueenCoordinates(x, y);
+    }
+
+    private final int x;
```

Comment:
In NQueens, it's not called x and y, but rowIndex and columnIndex.
For consistency, can we use the same naming?

Note that (x,y) starts from the top left of a screen - so using a different name might imply that there is some translation calculation when going from (x, y) to (rowIndex, columnIndex), which there is not I presume.